ext/io/console/console.c

```diff
@@ -1023,6 +1023,16 @@ console_goto(VALUE io, VALUE y, VALUE x)
     return io;
 }
 
+/*
+ * call-seq:
```

Comment:
I'm not sure how to document methods inside `#if defined _WIN32`. Do I need to duplicate it or is there an alternative way of doing it?